src/Utils/randomGenerator.js

```diff
@@ -0,0 +1,12 @@
+import { Random } from "@woowacourse/mission-utils";
+
+/**
+ *
+ * @returns {number}
+ * @summary 1부터 10까지의 랜덤한 숫자를 반환한다.
+ */
+const randomGenerator = () => {
+  return Random.pickNumberInRange(1, 10);
+};
```

Comment:
```suggestion
const randomGenerator = (min, max) => {
  return Random.pickNumberInRange(min, max);
};
```
이번 미션 요구사항이 0~9까지의 랜덤 숫자였던 것 같아요! 그리고 범위도 인자로 받으면 조금 더 범용성있는 함수가 될 것 같아요🫡

Reply:
헉... 이런.. 제대로 확인하지 못한 부분인 것 같네요... 하이고🥲  좋은 지적 감사합니다.
맞아요. 그리고 다른 분들 리뷰하면서 발견한 부분이기도 합니다! 제가 생각 외로 텍스트 숫자나 문자 그대로 사용한 경우가 많더라구요,, 다음 미션에서는 꼭 수정하고 싶은 부분이기도 합니다.

변수로 파라미터를 받아오니 더 직관적이기도 하네요. 유지보수를 생각해서라도 변수를 사용하는 습관을 들여야겠어요. 감사합니다